### PR TITLE
OSDOCS-10780: Style fix: Sample to example

### PR DESCRIPTION
**Pull request by jneczypor**
[OSDOCS-10780](https://issues.redhat.com//browse/OSDOCS-10780): Style fix: Sample to example. Change all instances of .Sample output to .Example output in the ROSA docs

Version(s):
4.15+

Issue:
https://issues.redhat.com/browse/OSDOCS-10780

Link to docs preview:

- https://76794--ocpdocs-pr.netlify.app/openshift-rosa/latest/cloud_experts_tutorials/cloud-experts-rosa-cloudwatch-sts.html
- https://76794--ocpdocs-pr.netlify.app/openshift-rosa/latest/rosa_architecture/rosa-oidc-overview.html
- https://76794--ocpdocs-pr.netlify.app/openshift-rosa/latest/rosa_architecture/rosa-sts-about-iam-resources.html
- https://76794--ocpdocs-pr.netlify.app/openshift-rosa/latest/rosa_hcp/rosa-hcp-creating-cluster-with-aws-kms-key.html
- https://76794--ocpdocs-pr.netlify.app/openshift-rosa/latest/rosa_hcp/rosa-hcp-sts-creating-a-cluster-quickly.html
- https://76794--ocpdocs-pr.netlify.app/openshift-rosa/latest/rosa_hcp/rosa-tuning-config.html
- https://76794--ocpdocs-pr.netlify.app/openshift-rosa/latest/rosa_planning/rosa-understanding-terraform.html
- https://76794--ocpdocs-pr.netlify.app/openshift-rosa/latest/support/troubleshooting/rosa-troubleshooting-iam-resources.html

No QE or PM needed.

Additional information:

### CODE REVIEW

File: cloud_experts_tutorials/rosa-mobb-prerequisites-tutorial.adoc [hunk 1]
```diff
@@ -107,7 +107,7 @@ $ aws configure
 $ aws configure
 ----
 +
-.Sample output
+.Example output
```
Comment (smunje1) [inline cloud_experts_tutorials/rosa-mobb-prerequisites-tutorial.adoc:110]: I am unable to see preview for this assembly. I can see see similar comment on Line 20 of this file, "//This file is not being built as of 2023-09-22 based on a conversation with Michael McNeill."

Is this expected?

File: cloud_experts_tutorials/rosa-mobb-prerequisites-tutorial.adoc [hunk 2]
```diff
@@ -127,7 +127,7 @@ $ aws sts get-caller-identity
 +
 You should receive output similar to the following:
 +
-.Sample output
+.Example output
```
Comment (smunje1) [inline cloud_experts_tutorials/rosa-mobb-prerequisites-tutorial.adoc:130]: I am unable to see preview for this assembly. I can see see similar comment on Line 20 of this file, "//This file is not being built as of 2023-09-22 based on a conversation with Michael McNeill."

Is this expected?

File: cloud_experts_tutorials/rosa-mobb-prerequisites-tutorial.adoc (hunk 1 above)
Comment (jneczypor) [reply cloud_experts_tutorials/rosa-mobb-prerequisites-tutorial.adoc:110]: Yes, some content is hidden on purpose. I still want to make the fixes for future release. Thanks!

File: cloud_experts_tutorials/rosa-mobb-prerequisites-tutorial.adoc (hunk 2 above)
Comment (jneczypor) [reply cloud_experts_tutorials/rosa-mobb-prerequisites-tutorial.adoc:130]: Yes, some content is hidden on purpose. I still want to make the fixes.

**Comment by bscott-rh** [comment]
> @jneczypor Nice work on your PR. Left a couple of comments for clarification. Everything else LGTM. As I am new to the peer review squad, I request that you wait until @bscott-rh gives approval.

Peer review LGTM.

**Review by jeana-redhat — APPROVED** [review]
LGTM

/remove-label merge-review-in-progress
/remove-label merge-review-needed

**Pull request merged**